docs/HowToFAQ/FAQModifiesOne.md

```diff
@@ -8,5 +8,6 @@ Is there a way to specify that all fields of an object, except a given one, don
 
 ## Answer
 
-Instead of writing `modifies this` or `modifies o`, you can write ``modifies this`f``
-to indicate that just the field `f` of `this` may be assigned to.
+Instead of writing `modifies this` or `modifies o`, you can write ``modifies this`f`` of ``modifies `f``
+to indicate that just the field `f` of `this` may be assigned to. Similarly you can use 
+``modifies a`f`` for fields of other objects.
```

Comment:
The first sentence uses both `this` and `o` as examples, but then the `o` is not one of the two alternatives later in the sentence. I suggest:

```suggestion
Instead of writing `modifies this` or `modifies o`, you can write ``modifies this`f`` (equivalently ``modifies `f``)
or ``modifies o`f``
to indicate that just the field `f` of `this` or `o`, respectively, may be assigned to.
```